ovs/extensions/healthcheck/arakoon.py

```diff
@@ -245,7 +250,11 @@ def _connection_worker(queue, result_handler):
 
     @classmethod
     @cluster_check
-    @expose_to_cli(MODULE, 'collapse-test', HealthCheckCLIRunner.ADDON_TYPE)
+    @expose_to_cli(MODULE, 'collapse-test', HealthcheckCLI.ADDON_TYPE,
+                   help='Verifies collapsing has occurred for all Arakoons',
+                   short_help='Test if Arakoon collapsing is not failing')
+    @expose_to_cli.option('--max_collapse-age', type=int, default=3, help='Maximum age in days for TLX')
```

Comment:
any specific reason for using both underscores and dashes?
I'd suggest to choose one (dashes are mostly used by you so......)